src/container/srv_container.c

```diff
@@ -4217,7 +4217,7 @@ cont_filter_match(struct rdb_tx *tx, struct cont *cont, daos_pool_cont_filter_t
 	int		rc = 0;
 
 	/* logical OR combining: start with false result, transition to true on first match */
-	if ((filt->pcf_parts > 0) && (combine_op == PCF_COMBINE_LOGICAL_OR))
+	if ((filt->pcf_parts != 0) && (combine_op == PCF_COMBINE_LOGICAL_OR))
```

Comment:
This one I'm not sure about but pcf_parts appears to be a pointer, in which case I think this is the correct change.

Reply:
If it's a pointer, then should it check NULL instead of 0?

Reply:
Will change (and the same change in dfs) when I re-push to remove the change to landings.yaml.